### ISSUE TITLE
[BUG] Random Adjective not applying to tokens

### ISSUE THREAD

**Issue by fiddlest1x** [Moerill/token-mold#216]
**Using the template**  
Choose a fitting  title to your issue.  
Please go through each section, adding as much detailed information as possible and remove the default text. If a section is completly irrelevant for some reason, delete it completly.  
Also ideally be somewhat responsive in this issues thread so you can answer possible questions i have.  

**Describe the bug**
When I set random adjectives to apply to tokens, none apply. I have a random table that I use and I've tried the default ones. Adding counting numbers works, but nothing else. The below pictures are on Foundry 12.331 and Dnd 3.3.1. It does not work on my Forge either with dnd 4.1. Additionally, it seems to ignore the overwrite settings for Displaying the token name to the owner when I have the random adjectives on.

Also post console errors (F12 for most browsers) here that occur.
I get this when I try to place a token
![image](https://github.com/user-attachments/assets/faebd05e-4c1a-4a4a-bbcd-68d48114f997)

I get this when I save&close token mold settings
![image](https://github.com/user-attachments/assets/0bbaf35f-042c-4ad7-b663-585814fcd081)

**Desktop (please complete the following information):**
 - OS: Win11 23H2
 - Browser Chrome and Foundry VTT offline
 - FVTT Version  12.331
 - Module version 2.22.3

**Additional context**

**Comment by fiddlest1x** [comment]
Yes sir. On my offline Foundry it's just Dnd 3.3.1 and no other modules
active. I realize I haven't tried outside Dnd so I'll do that and get back
to you.

**Comment by fiddlest1x** [comment]
Checked again. Works on a fresh world of any system including dnd. Even
with all modules disabled in my game, however, it throws up the errors I
took a pic of.

**Comment by fiddlest1x** [comment]
Sorry for the emails, but I think I sorta got something. I removed the
settings.db and settings folders from my appdata locally. It seems it's
remembering things even if I uninstall the module and it's causing those
issues. Removing them caused it to work as normal. I have a workaround, I
hope it doesn't screw anything else, of just downloading my world, removing
that folder, and reuploading it.

**Comment by BeardedGnome** [comment]
I may have inadvertently made an incompatible change to the way settings are saved. I'll look closer to see if I can isolate it... eventually.

**Comment by fiddlest1x** [comment]
Lol thanks. I appreciate all your work. Regardless of when or if you do it.

**Comment by BeardedGnome** [comment]
Closing this with the release of 2.23.1. Please let me know if you continue to have issues.
